git.md

```diff
@@ -55,9 +64,9 @@
 
 ## Commit Messages
 
-- You have two options to choose from:
+- You have two options to choose from
 
-### Conventional Commits
+#### Conventional Commits
```

Comment:
There is an H4 heading instead of H3. The same for other headings in the document.